addons/website_version/controllers/main.py

```diff
@@ -0,0 +1,155 @@
+# -*- coding: utf-8 -*-
+import datetime
+from openerp import http
+from openerp.http import request
+from openerp.addons.website.controllers.main import Website
+
+
+class Versioning_Controller(Website):
+
+    @http.route(['/website_version/change_version'], type='json', auth="user", website=True)
+    def change_version(self, version_id):
+        request.session['version_id'] = version_id
+        return version_id
+
+    @http.route(['/website_version/create_version'], type='json', auth="user", website=True)
+    def create_version(self, name, version_id):
+        if not name:
+            name = datetime.datetime.now().strftime('%Y-%m-%d %H:%M:%S')
```

Comment:
Should'nt we use DEFAULT_SERVER_DATETIME_FORMAT defined in tools ? It seems it is just for naming a version, but if it has no side-effect, let's use what the admin wants to use.